mlrun/datastore/targets.py

```diff
@@ -28,6 +28,7 @@
 from mlrun.config import config
 from mlrun.model import DataSource, DataTarget, DataTargetBase, TargetPathObject
 from mlrun.utils import now_date
+from mlrun.utils.helpers import default_time_partitioning_granularity, legal_time_units
```

Comment:
we are trying to avoid using `from x import y`, I know there are already few imports using `from` but let's try to avoid it when possible when adding new imports.